kamu-core/tests/tests/test_query_service_impl.rs

```diff
@@ -138,8 +140,8 @@ async fn create_catalog_with_s3_workspace(s3: &S3) -> dill::Catalog {
         .add_value(infra::ObjectStoreBuilderLocalFs::new())
         .bind::<dyn ObjectStoreBuilder, infra::ObjectStoreBuilderLocalFs>()
         .add_value(infra::ObjectStoreBuilderS3::new(
-            s3_context.bucket,
-            endpoint.unwrap(),
+            s3_context,
+            s3_credentials,
```

Comment:
Do we need to pass credentials separately if they can already be obtained from `S3Context`?

Reply:
Yes, because it's async operation, and the context around is synchronous